File: abc/abc279/b.cpp
// https://atcoder.jp/contests/abc279/submissions/36799730

#include <bits/stdc++.h>
using namespace std;

int main() {
  cin.tie(0);
  ios_base::sync_with_stdio(0);

  string s, t;
  cin >> s >> t;

  if(t.size() > s.size()) {
    cout << "No";
    return 0;
  }

  for(int i = 0; i < (s.size() - t.size() + 1); i++) {
    for(int j = 0; j < t.size(); j++) {
      if(s[i + j] != t[j]) break;

      if(j == t.size() - 1) {
        cout << "Yes";
        return 0;
      }
    }
  }

  cout << "No";
}

File: abc/abc279/b.cs
// https://atcoder.jp/contests/abc279/submissions/46445913

using System;

class A
{
  static void Main()
  {
    string s = Console.ReadLine();
    string t = Console.ReadLine();

    if(t.Length > s.Length)
    {
      Console.WriteLine("No");
      return;
    }

    bool ans = false;
    for(int i = 0; i <= s.Length - t.Length; i++)
    {
      bool check = true;
      for(int j = 0; j < t.Length; j++)
      {
        if(s[i + j] != t[j])
        {
          check = false;
          break;
        }
      }

      if(check)
      {
        ans = true;
        break;
      }
    }

    if(ans)
    {
      Console.WriteLine("Yes");
    }
    else
    {
      Console.WriteLine("No");
    }
  }
}

